libs/SmartStore/SmartStore/Classes/SFSmartStore.h

```diff
@@ -295,15 +296,15 @@ extern NSString *const EXPLAIN_ROWS;
  @return YES if the soup is registered or already exists.
 
  */
-- (BOOL)registerSoupWithSpec:(SFSoupSpec*)soupSpec withIndexSpecs:(NSArray*)indexSpecs error:(NSError**)error;
+- (BOOL)registerSoupWithSpec:(SFSoupSpec*)soupSpec withIndexSpecs:(NSArray*)indexSpecs error:(NSError**)error NS_SWIFT_NAME(registerSoup(soupSpec:indexSpecs:));
 
 /**
  Get the number of entries that would be returned with the given query spec
  
  @param querySpec A native query spec.
  @param error Sets/returns any error generated as part of the process.
  */
-- (NSUInteger)countWithQuerySpec:(SFQuerySpec*)querySpec error:(NSError **)error;
+- (NSNumber* __nullable) countWithQuerySpec:(SFQuerySpec*)querySpec error:(NSError **)error NS_SWIFT_NAME(count(querySpec:));
```

Comment:
Changing return type to a (nullable) object so that the swift interface throws an error.